code-scanning/sbt-dependency-graph.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# More details at https://github.com/scalacenter/sbt-dependency-graph-action 
+
+name: Sbt Dependency Graph
+
+on:
+  push:
+    branches: [ $default-branch ]
+
```

Comment:
Kindly add workflow level permission needed by github token. Refer this: https://github.com/actions/starter-workflows/blob/main/code-scanning/checkmarx.yml#L20